### ISSUE TITLE
Delay when Equipping Gear

### ISSUE THREAD

**Issue by Laserwolve** [Laserwolve-Games/DaggerQuest#36]
See #35. That issue was resolved by adding an arbitrary fixed wait time, to allow the incoming gear to fully 'spawn'. We are now left with a solid second wait between the action that should equip a gear, and being able to visibly see it. This 1 second wait might be too much on higher end machines but too little on weaker machines.

Ultimately, we don't have a way to know when a sprite of this size is actually visible.

**Comment by Laserwolve** [comment]
I previously attributed this solely to gear size. However, I ran a test in a blank project with 11 GBs of sprites on a layout and no events and this stuttering wasn't present.

https://github.com/Laserwolve-Games/DaggerQuest/assets/87336074/1cb7c3f0-5e77-46cb-b9c7-7bb2f2361ab2

I tried ripping events out, I got down to about ~60 events and I was only able to recreate the stuttering/flashing about 10% of the time.
